src/components/wallet/TokenBox.js

```diff
@@ -62,7 +74,11 @@ const TokenBox = ({ token }) => {
             </div>
             <div className='desc'>
                 <span>{token.symbol}</span>
-                <span title={token.contract}>{token.contract}</span>
+                <span title={token.contract}>
+                    <a href={`https://explorer.near.org/accounts/${token.contract}`} target='_blank' rel='noopener noreferrer'>
```

Comment:
@vgrichina I hard coded the explorer URL since you said the contracts are currently only on mainnet, otherwise the links will be broken on testnet wallet

Reply:
@Patrick1904 we have `EXPLORER_URL` in wallet.js, no need to hardcode

> since you said the contracts are currently only on mainnet, otherwise the links will be broken on testnet wallet

that was about specific whitelisted contracts, we can whitelist different ones in config for testnet